airbyte-webapp-e2e-tests/cypress/pages/replicationPage.ts

```diff
@@ -8,20 +8,22 @@ const destinationNamespaceDefault = "div[data-testid='namespaceDefinition-destin
 const destinationNamespaceSource = "div[data-testid='namespaceDefinition-source']";
 const destinationNamespaceCustomInput = "input[data-testid='input']";
 const syncModeDropdown = "div[data-testid='syncSettingsDropdown'] input";
-const cursorFieldDropdown = "button[class^='PathPopoutButton_button']";
-const streamFieldNames = "[class^='TreeRowWrapper_rowWrapper'] span";
-const streamDataTypes = "[class^='TreeRowWrapper_rowWrapper'] div:nth-child(2)";
-const ExpandStreamDetailsTableBtn = "[class^='Arrow_container__']";
-const cursorFieldText = "[class^='PathPopoutButton_button__']";
-const primaryKeyText = "[class^='PathPopoutButton_button__']";
+const getFieldDropdownContainer = (streamName: string, type: Dropdown) => `div[id='${streamName}_${type}_pathPopout']`;
+const getFieldDropdownButton = (streamName: string, type: Dropdown) =>
+  `button[data-testid='${streamName}_${type}_pathPopout']`;
+const getFieldDropdownOption = (value: string) => `div[data-testid='${value}']`;
+const dropDownOverlayContainer = "div[data-testid='overlayContainer']";
+const streamNameCell = "[data-testid='nameCell']";
+const streamDataTypeCell = "[data-testid='dataTypeCell']";
+const getExpandStreamArrowBtn = (streamName: string) => `[data-testid='${streamName}_expandStreamDetails']`;
 const preFilledPrimaryKeyText = "div[class^='PathPopout_text']";
```

Comment:
classnames make the Vite build fail, can we use a test-id on this? (Context: [Slack thread](https://airbytehq-team.slack.com/archives/C046TSAN606/p1673965508898039))

Reply:
Oh, okay, will fix that